Frameworks/include/CoreGraphics/DWriteWrapper.h

```diff
@@ -75,4 +76,9 @@ COREGRAPHICS_EXPORT HRESULT _DWriteFontGetBoundingBoxesForGlyphs(
 // DWrite functions relating to font binary data
 COREGRAPHICS_EXPORT HRESULT _DWriteRegisterFontsWithDatas(CFArrayRef fontDatas, CFArrayRef* errors);
 COREGRAPHICS_EXPORT HRESULT _DWriteUnregisterFontsWithDatas(CFArrayRef fontDatas, CFArrayRef* errors);
-COREGRAPHICS_EXPORT HRESULT _DWriteCreateFontFaceWithData(CGDataProviderRef data, IDWriteFontFace** outFontFace);
\ No newline at end of file
+COREGRAPHICS_EXPORT HRESULT _DWriteCreateFontFaceWithData(CGDataProviderRef data, IDWriteFontFace** outFontFace);
+
+inline uint32_t _CTToDWriteFontTableTag(uint32_t tag) {
+    // CT has the opposite byte order of DWrite, so we need 'BASE' -> 'ESAB'
```

Comment:
A dev can pass in a specific tag for CTFontCopyTable though. We'd at least need to keep the function around (though I guess we could reverse the internal constants)